hack/openstack-populator/Containerfile

```diff
@@ -0,0 +1,13 @@
+FROM registry.access.redhat.com/ubi8/go-toolset:1.18.9-8 as builder
+ENV GOPATH=$APP_ROOT
+
+COPY go.mod ./
+COPY go.sum ./
+COPY cmd/openstack-populator/openstack-populator.go ./
+RUN go mod download
+RUN go build -o openstack-populator
+
+FROM registry.access.redhat.com/ubi9-minimal:9.1.0-1760.1675784957
```

Comment:
I think we should use the centos minimal version same as on the ovirt populator.
`FROM quay.io/centos/centos:stream9-minimal`

Reply:
In ovirt it was used because it needed some specific dependencies that weren't available, no @ahadas ?
That said, I don't mind changing it